packages/bls12-381/bls12-381.18.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "3.0" }
+  "ocaml" { >= "4.14" }
+  "integers"
+  "integers_stubs_js"
+  "zarith" { >= "1.12" & < "1.13" }
+  "zarith_stubs_js"
+  "hex" { >= "1.3.0" }
+  "tezt" { with-test & >= "3.1.1" }
+  "octez-alcotezt" { with-test & = version }
+  "qcheck-alcotest" { with-test & >= "0.20" }
+]
+build: [
+  ["rm" "-r" "vendors" "contrib"]
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
```

Comment:
```suggestion
]
available: arch != "arm32" & arch != "x86_32"
```

Reply:
It's still not working on ppc64 and s390x, so this does need some refinement. But this can be fixed post-merge as well.